parlai/mturk/core/mturk_manager.py

```diff
@@ -26,8 +26,21 @@
 # Timeout before cancelling a world start
 WORLD_START_TIMEOUT = 11
 
-# Multiplier to apply when creating hits to ensure worker availibility
-HIT_MULT = 1.5
+# Multiplier to apply when creating hits to ensure worker availibility. As the
+# number of HITs increases, this decreases
+HIT_MULT_SCALE = [
```

Comment:
Are these arbitrary multipliers/cutoffs? They look good to me, just curious

Reply:
Yeah they're mostly derived from a rule of thumb based on claim rates and initial disconnect rates. More than entirely arbitrary, but way less than scientifically concrete.

Reply:
ok good to know it's grounded on some empirical evidence